src/Kunstmaan/AdminListBundle/Resources/translations/messages.de.yml

```diff
@@ -13,8 +13,8 @@ kuma_admin_list:
     move_up: Move up
     move_down: Move down
 filter:
-  "true": true
-  "false": false
+  true: true
```

Comment:
@Numkil did some fixes for this in #1909, so we should check if we can merge it this way or revert these kind of changes

Reply:
Yes, when the PR of @Numkil is merged, 5.0 branch should be upmerged, and crowdin should pick up this translations.